community/gill-supabase-web3/src/app/account/[address]/page.tsx

```diff
@@ -0,0 +1,12 @@
+'use client'
```

Comment:
You can push down the `use client` directive further down the tree of components.